src/test/java/org/broadinstitute/hellbender/tools/gvs/common/WeightedSplitIntervalsIntegrationTest.java

```diff
@@ -88,6 +79,41 @@ public void testNoLossRealisticWgs() {
 
     }
 
+    @Test
+    public void testHandleZeroBasesToTake() {
```

Comment:
If you ran this test with the pre-fix code does it fail?

Reply:
Yes that's how I discovered all the scatter widths that exercise the condition. Much easier to find if the code under test throws an exception, no need to set a breakpoint. 🙂